### ISSUE TITLE
PhET-iO production versions due "late August"

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/tasks#1040]
At 7/2/2020 phet-io meeting, @kathy-phet presented scheduling information that she received from the client.

At 7/2/2020 dev meeting, we noted that there are several sims (ph-scale, EFAC, SOM, ... others?) that all have the same "late August" milestone for delivering PhET-iO production versions.  Instead of tackling each sim independently, a coordinated effort seems like a good idea, so that we don't duplicate work, and so that the client gets a similar version of Studio and other wrappers.

This will be discussed at 7/9/2020 phet-io meeting, with these goals:

* translate "late August" to a date
* identify which sims are included
* identify what common-code work needs to be completed (PhET-iO, blocking issues,...)
* identify what sim-specific work needs to be completed (client change requests, issues prerequisite for RC,...)
* put together a schedule

**Comment by samreid** [comment]
Self unassigning because I am unlikely to work on this issue before Thursday's PhET-iO meeting, where this issue is marked on the agenda for discussion.

**Comment by pixelzoom** [comment]
The scheduling info that @kathy-phet received from the client included:

> Natural Selection (we don't expect before June 2021, but will use whenever ready)

This date is very surprising. Our original milestone (which we're not going to meet) was August 1, 2020. @kathy-phet Is June 2021 really correct?

**Comment by zepumph** [comment]
From phet-io meeting today:

>translate "late August" to a date

@kathy-phet is meeting with the client sometime next week. She will ask for a specific deadline.

>identify which sims are included

She thinks we should plan on the complete list being ph-scale, EFAC, SOM, but will update us if that changes from the meeting.

>identify what common-code work needs to be completed (PhET-iO, blocking issues,...)

We have worked down the list of blocking publication issues, such that all that remains would be anything that comes from sim-specific phet-io design requirements.

>identify what sim-specific work needs to be completed (client change requests, issues prerequisite for RC,...)

That seems good to do! Likely issues should be made in each repo:
* ph-scale: https://github.com/phetsims/ph-scale/issues/166
  * @pixelzoom - this sim is ready to go!
* states-of-matter: @jbphet to create and link an issue to identify the sim steps needed for phet-io publication
  * @jbphet said that there are a few things to do before taking shas
    * @kathy-phet: how much work is that?
    * @jbphet: ~24 hours of time to fix open issues
    * Other than getting number line feature complete (perhaps just tomorrow), @jbphet will be all hands on deck for SOM publishing. He guesses it will be ready for RC by July 24th.
* energy-forms-and-changes: @chrisklus to create and link an issue to identify the sim steps needed for phet-io publication
  * @chrisklus estimates that remaining bugs (to get to RC) would take 8-16 hours of work.
* gravity-and-orbits:
  * @samreid: only 1 issue left before it is ready for a second (spot check?) dev test.
  * @samreid: there are multiple, deeper issues that need to be done before RC. 16-32 hours or so, depending on how many "optional" improvement issues are done before RC.

>Natural Selection (we don't expect before June 2021, but will use whenever ready)

@kathy-phet was also surprised to hear this date, as in the NS contract it states an "estimated date" of August 1st. Current goal is end of august for a initial dev delivery (lower priority of publishing the above releases to production). @kathy-phet will ask at the aforementioned client meeting when in the year our client will want to use the sim. She will get a milestone date for the phet-io dev version. From that we will set a "prototype/preview" publication milestone.



Ideally we would want to reuse shas, we discussed this, among other QA priorities like MAL RC, and decided to create RC shas for ph-scale and MAL now.

EFAC and SOM both seem to be on a similar timeline of about 2 weeks, and we will touch base with them together once they are closing to being ready for RC shas.

**Comment by jbphet** [comment]
I'll be working on individual issues to get SOM out, and all are logged in GitHub, so I'm unassigning myself from this for now.

**Comment by pixelzoom** [comment]
pH Scale RC is in progress, proceeding on its own schedule.  So unassigning myself.

**Comment by zepumph** [comment]
I feel like at this point this issue is stale. We are past august, and timeline has been communicated to us moving forward. I'm going to close this issue, but please reopen if anyone has more to say.
